share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/fieldBackground.param

```diff
@@ -84,6 +86,39 @@ namespace picongpu
         /* Add this additional field? */
         static constexpr bool activated = true;
 
+        /* This setup is based on [Taflove, Hagness], section 7.11.1.
+         * The difference is we consider both 2D and 3D cases, and grid size may be increased due to our absorber being
+         * part of the simulation area, not located outside of it as in the book.
+         *
+         * Example of a rectangular conductor with a steady current.
+         *
+         * The conductor is oriented along the an axis.
+         * It shall have a diameter of 32 cells.
```

Comment:
In my opinion, "diameter" is not the right word for a rectangularly shaped wire. How about edge length?

Reply:
Starting from line 96, the whole documentation is actually outdated and not referring to the Taflove example. This text is left over from a different example I once made and from which I derived for the Taflove example :sweat_smile:
Starting from line 95 to 110, the comment should read
```
The conductor is oriented along the y-axis.
Its edge length can be adjusted by the variable halfWidth in order to apply the test with meaningful results to higher-order solvers, too.

In accordance with Taflove, the current in the wire ramps up over time according to a differentiated Gaussian.
This defines the current density amplitude, too.
Therefore, the total current through the wire scales with the wire's halfWidth.
```
or similar.